<launch>

    <node pkg="navigation_system" type="UWBSystem_Node" name="UWBSystem_Node" output="screen" >
    	<param name="imu_enable" type="bool" value="true" />
    	<param name="mag_enable" type="bool" value="true" />
    	<param name="attitude_enable" type="bool" value="true" />
    	<param name="log_file" type="bool" value="false" />
    	<param name="freq_filter" type="int" value="40" />

    	<!-- anchor 0 -->
    	<param name="anchor0_X" type="double" value="0.0" />
    	<param name="anchor0_Y" type="double" value="0.0" />
    	<param name="anchor0_Z" type="double" value="0.0" />
    	<!-- anchor 1 -->
    	<param name="anchor1_X" type="double" value="0.0" />
    	<param name="anchor1_Y" type="double" value="0.0" />
    	<param name="anchor1_Z" type="double" value="0.0" />
    	<!-- anchor 2 -->
    	<param name="anchor2_X" type="double" value="0.0" />
    	<param name="anchor2_Y" type="double" value="0.0" />
    	<param name="anchor2_Z" type="double" value="0.0" />
    	<!-- anchor 3 -->
    	<param name="anchor3_X" type="double" value="0.0" />
    	<param name="anchor3_Y" type="double" value="0.0" />
    	<param name="anchor3_Z" type="double" value="0.0" />
    </node>

</launch>
